### PR TITLE
Update K8s versions in example files

### PR DESCRIPTION
**Pull request by timuthy**
**What this PR does / why we need it**:
This PR updates the K8s versions quoted in examples to the latest ones.

```improvement operator
NONE
```

### CODE REVIEW

**Review by zanetworker — DISMISSED** [review]

File: example/30-cloudprofile-alicloud.yaml [hunk 1]
```diff
@@ -15,9 +15,9 @@ spec:
       - name: unmanaged
       kubernetes:
         versions:
-        - 1.12.1
-        - 1.11.3
-        - 1.10.8
+        - 1.12.3
+        - 1.11.5
+        - 1.10.11
```
Comment (zanetworker) [inline example/30-cloudprofile-alicloud.yaml:20]: is there a reason why the comment `# - 1.9.11 # Usage is not recommended, see CVE-2018-1002105` was not added here as well?

Comment (rfranzke) [reply example/30-cloudprofile-alicloud.yaml:20]: +1

Comment (timuthy) [reply example/30-cloudprofile-alicloud.yaml:20]: The reason is that 1.9.x has never been part of the Alicloud Cloudprofile and I didn't want to question that in this PR. I can still add it, though.

Comment (rfranzke) [reply example/30-cloudprofile-alicloud.yaml:20]: I would add it for the sake of uniformity

**Comment by timuthy** [comment]
@rfranzke, @zanetworker  added 1.9.11 also for Alicloud.

**Review by rfranzke — APPROVED** [review]
/lgtm

**Pull request merged**